packages/react-native-icons/src/utils/fonts/createFluentFontIcon.tsx

```diff
@@ -26,89 +31,83 @@ export const enum FontFile {
     Light = 3
 }
 
-const FONT_FAMILY_MAP = {
-    [FontFile.Filled]: 'FluentSystemIconsFilled',
-    [FontFile.Regular]: 'FluentSystemIconsRegular',
-    [FontFile.Resizable]: 'FluentSystemIcons',
-} as const;
-
-const useStaticStyles = makeStaticStyles(`
-@font-face {
-    font-family: ${FONT_FAMILY_MAP[FontFile.Filled]};
-    src: url(${JSON.stringify(fontFilledWoff2)}) format("woff2"),
-    url(${JSON.stringify(fontFilledWoff)}) format("woff"),
-    url(${JSON.stringify(fontFilledTtf)}) format("truetype");
-}
-@font-face {
-    font-family: ${FONT_FAMILY_MAP[FontFile.Regular]};
-    src: url(${JSON.stringify(fontRegularWoff2)}) format("woff2"),
-    url(${JSON.stringify(fontRegularWoff)}) format("woff"),
-    url(${JSON.stringify(fontRegularTtf)}) format("truetype");
-}
-
-@font-face {
-    font-family: ${FONT_FAMILY_MAP[FontFile.Light]};
-    src: url(${JSON.stringify(fontLightWoff2)}) format("woff2"),
-    url(${JSON.stringify(fontLightWoff)}) format("woff"),
-    url(${JSON.stringify(fontLightTtf)}) format("truetype");
-}
-
-@font-face {
-    font-family: ${FONT_FAMILY_MAP[FontFile.Resizable]};
-    src: url(${JSON.stringify(fontOneSizeWoff2)}) format("woff2"),
-    url(${JSON.stringify(fontOneSizeWoff)}) format("woff"),
-    url(${JSON.stringify(fontOneSizeTtf)}) format("truetype");
-}
-`)
-
-const useRootStyles = makeStyles({
-    root: {
-        display: 'inline-block',
-        fontStyle: 'normal',
-        lineHeight: '1em',
-
-        "@media (forced-colors: active)": {
-            forcedColorAdjust: 'auto',
+// const FONT_FAMILY_MAP = {
+//     [FontFile.Filled]: 'FluentSystemIconsFilled',
+//     [FontFile.Regular]: 'FluentSystemIconsRegular',
```

Comment:
Remove all the commented code?

Reply:
I can, I was going to leave it for later for when we can do font icon for RN, but I guess I can just look at what the react-icons package has